docker-compose.ci.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    environment:
+      DB_PASSWORD: ${DB_PASSWORD:-password}
+      DB_HOST: ${DB_HOST:-pg}
+      DB_PORT: ${DB_PORT:-5432}
+      DB_USER: ${DB_USER:-postgres}
+      DB_NAME: ${DB_NAME:-dev-directory}
+    ports:
+      - 3000:3000
+    depends_on:
+        - pg
```

Comment:
Indentation is off here.

Reply:
Corrected this and removed the redundant ENV variables.